src/components/forums/forum/ForumContent.tsx

```diff
@@ -216,6 +225,43 @@ export function ForumContent(props: ForumContentProps) {
     }
   };
 
+  const deleteAccessToken = async () => {
+    setRemoveAccessToken({ ...removeAccessToken, removing: true });
+    try {
+      const tx = await forumObject.deleteForumPostRestriction(
```

Comment:
Currently, deleteForumPostRestrictions remove ALL restrictions on a post. 

When removing a single accessToken among a list, the proper way to "delete" would be to filter out the token being removed, then set forum restriction to the remaining list of tokens using `nftListAnyOwnership` field in `PostRestriction` type. Otherwise, all post restrictions can be removed using `deleteForumPostRestriction`. 

It's probably best to move this logic into postboxWrapper with an additional optional parameter of the token being removed.